### ISSUE TITLE
Add portable GUI client for PC

### ISSUE THREAD

**Issue by Cyberavater** [RikkaApps/Shizuku#63]
It'd nice n easy to have a PC GUI client.

**Comment by jerrdk** [comment]
Good

**Comment by jerrdk** [comment]
> It'd nice n easy to have a PC GUI client.

Code pin cc

**Comment by jerrdk** [comment]
Give me money cc

Pada tanggal Sab, 16 Mei 2020 18:30, Cyberavater <notifications@github.com>
menulis:

**Comment by RikkaW** [comment]
Maybe use WebUSB is a more elegant choice?

**Comment by Cyberavater** [comment]
IDK, does WebUSB support ADB over wifi? (In case the user want to use it via wifi)

**Comment by ryn-inaba** [comment]
I'm working on a GUI to start Shizuku. I can put it on Github when I finished

**Comment by ryn-inaba** [comment]
I pushed the initial commit and pushed a pre-release binary. [The binary can be found here](https://github.com/ryn-inaba/ShizukuStarter/releases) and [the code here](https://github.com/ryn-inaba/ShizukuStarter). Let me know if there's any bugs or anything that should be added

**Comment by Cyberavater** [comment]
I'm out of town, so I can't really test it here. 😔

**Comment by ppoffice** [comment]
Here is a live demo/prototype: https://z5uh2.csb.app/

Steps:
- Open the demo in a Chromium-based browser (one of the browsers listed here https://caniuse.com/#search=webusb).
- Plug your Android device to your PC via USB. Make sure the ADB debugging is enabled and the ADB server is stopped on your PC.
- Press the "Connect" button.
- The web app should start Shizuku service for you automatically.

Note: It does not support ADB over Wi-Fi.
Hope this will help the Shizuku Team implement Web GUI.

**Comment by Cyberavater** [comment]
Yes, after finishing the app, it can be uploaded to chrome web store like one this;
https://chrome.google.com/webstore/detail/helium-backup/gpglbgbpeobllokpmeagpoagjbfknanl

**Comment by coder0107git** [comment]
[WebADB](https://github.com/yume-chan/ya-webadb) (based on WebUSB) might be helpful, or at least useful for inspiration. The official demo seems to be [Tango](https://app.tangoapp.dev/).
